src/dom/Value.ts

```diff
@@ -138,6 +138,12 @@ export interface Value {
      * @param writer    An Ion Writer to which the value should be written.
      */
     writeTo(writer: Writer): void;
+
+    /**
+     * For the Struct type, deletes the key/field if it's in the _fields collection
+     * otherwise throws an Error.
```

Comment:
Two minor thoughts here:
* This is a typedoc comment (like Javadoc, but for Typescript) and will appear in [the public-facing documentation](https://amzn.github.io/ion-js/api/interfaces/_dom_value_.value-1.html#fieldnames) that's generated during the release. Folks reading that won't be aware of the `_fields` collection, which is an implementation detail in the `Struct` class.
* While the field name is basically a key, I suggest using either the Ion terminology ("field name" + "field value") or the JS terminology ("property name" + "property value") for consistency.

```suggestion
     * For the Struct type, deletes the field with provided name if present. Returns a boolean 
     * indicating whether the Struct was modified. For all other types, throws an Error.
```